spring-cloud-kubernetes-fabric8-discovery/src/main/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesDiscoveryClient.java

```diff
@@ -257,16 +258,6 @@ private int findEndpointPort(EndpointSubset s, String serviceId, String primaryP
 		}
 	}
 
-	private EndpointSubsetNS getSubsetsFromEndpoints(Endpoints endpoints) {
```

Comment:
move this method to utils. I'd like to "chop" DiscoverClient a lot more and individually test it and make it easier to read and work with